ui/admin/tests/acceptance/roles/list-test.js

```diff
@@ -122,4 +125,17 @@ module('Acceptance | roles | list', function (hooks) {
     assert.dom(`[href="${urls.role2}"]`).doesNotExist();
     assert.dom(NO_RESULTS_MSG_SELECTOR).includesText('No results found');
   });
+
+  test('correct badge in grants applied column is visible to user', async function (assert) {
+    instances.role1.grant_scope_ids = instances.role1.grant_scope_ids.filter(
+      (id) => id !== GRANT_SCOPE_THIS,
+    );
+    await visit(urls.orgScope);
+
+    await click(`[href="${urls.roles}"]`);
+
+    assert.true(instances.role2.grant_scope_ids.includes(GRANT_SCOPE_THIS));
+    assert.dom(ROLE_BADGE_SELECTOR(instances.role1.id)).hasText('No');
+    assert.dom(ROLE_BADGE_SELECTOR(instances.role2.id)).hasText('Yes');
```

Comment:
non-blocking: I think we can test our tooltips here as well (or a separate test) using [focus()](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#focus) from `@ember/test-helpers`

Reply:
ahh great suggestion, ill add that in

Reply:
Nice!